### PR TITLE
Partially revert "Create a new namespace for openstack-k8s-operators"

### PR DESCRIPTION
**Pull request by viroel**
Partially reverts openshift/release#35052

### CODE REVIEW

File: core-services/sync-rover-groups/_config.yaml [hunk 1]
```diff
@@ -77,7 +77,6 @@ groups:
     - app.ci
   openstack-k8s-operators-ci:
     clusters:
-    - app.ci
```
Comment (hongkailiu) [inline core-services/sync-rover-groups/_config.yaml:80]: You still need the group on app.ci if you want to access the images on app.ci.

Comment (viroel) [reply core-services/sync-rover-groups/_config.yaml]: ack

**Review by hongkailiu — APPROVED** [review]
Related to https://github.com/openshift/release/pull/36812

I will delete those rolebindings after merging:

```console
oc get rolebinding -n openstack-k8s-operators admin image-puller
NAME           ROLE                              AGE
admin          ClusterRole/admin                 57d
image-puller   ClusterRole/system:image-puller   57d
```

**Comment by hongkailiu** [comment]
```console
oc delete rolebinding -n openstack-k8s-operators admin image-puller --as system:admin
rolebinding.rbac.authorization.k8s.io "admin" deleted
rolebinding.rbac.authorization.k8s.io "image-puller" deleted
```

**Pull request merged**